serviceregistry-server/src/main/java/no/difi/meldingsutveksling/serviceregistry/client/brreg/BrregClientImpl.java

```diff
@@ -45,13 +45,18 @@ public BrregClientImpl(URI uri) {
      */
     @Override
     public Optional<BrregEnhet> getBrregUnderenhetByOrgnr(String orgnr) {
-        return getEnhet("enhetsregisteret/underenhet", orgnr);
+        return getEnhet("enhetsregisteret/api/underenheter/", orgnr);
     }
 
     private Optional<BrregEnhet> getEnhet(String registerUriPart, String orgnr) {
         URI currentURI = uri.resolve(String.format("%s/%s.json", registerUriPart, orgnr));
 
         RestTemplate rt = new RestTemplate();
+        HttpHeaders header = new HttpHeaders();
+        header.set("Accept", "application/vnd.brreg.enhetsregisteret.v1+json");
```

Comment:
Det er forskjellig versjonering for enheter- og underenehter-oppslagene.

Reply:
Skal legge til for begge